includes/admin/meta-boxes/class-wc-meta-box-order-data.php

```diff
@@ -344,6 +344,8 @@ public static function output( $post ) {
 
 								if ( 'billing_phone' === $field_name ) {
 									$field_value = wc_make_phone_clickable( $field_value );
+								} else if ( 'billing_email' === $field_name ) {
+									$field_value = '<a href="mailto:' . $field_value . '">' . $field_value . '</a>';
```

Comment:
```suggestion
									$field_value = '<a href="' . esc_url( 'mailto:' . $field_value ) . '">' . $field_value . '</a>';
```

Reply:
WTH! I did escape...not sure why this got pushed up!! Also the $field_value should be escaped too..

Reply:
Fixed...another round plz.